### PR TITLE
Add paging to transaction events

### LINKED ISSUE #14: Simple transfers from contracts should be paged
**Description**
`simple_transfer`s from smart contracts should be paged similarly to how regular transactions are paged.
The unpaged behaviour becomes an issue for contracts with large action trees that produce simple transfers.

Example of current behaviour:
image

### PR DESCRIPTION
**Pull request by limemloh**
## Purpose

Closes #14

## Changes

- Minor refactor to Paging module splitting pager and visibleItems into two functions.
- Add paging to transaction events in block explorer and the transaction lookup page.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by limemloh** [comment]
Block with a lot of events on testnet
https://dashboard.testnet.concordium.com/chain/a4e733fd2929047cdc69d29ad738293010d779185eca1997aaae46346cb30216

File: src/elm/Main.elm [hunk 1]
```diff
@@ -92,6 +92,7 @@ type Msg
     | HideToast Int
     | StorageDocReceived D.Value
     | SetCookieConsent Bool
+    | Never
```
Comment (Bargsteen) [inline src/elm/Main.elm:95]: I think some documentation on why this variant is needed would be good.

File: src/elm/Main.elm [hunk 2]
```diff
@@ -601,8 +605,8 @@ translateMsg msg =
         Explorer.View.UrlClicked link ->
             UrlClicked link
 
-        Explorer.View.TransactionPaging pagingMsg ->
-            ExplorerMsg <| Explorer.TransactionPaging pagingMsg
+        Explorer.View.Never ->
+            Never
```
Comment (Bargsteen) [inline src/elm/Main.elm:609]: Same here.

**Review by Bargsteen — CHANGES_REQUESTED** [review]
Nice work, just missing a bit of documentation.

File: src/elm/Main.elm (hunk 1 above)
Comment (limemloh) [reply src/elm/Main.elm]: I have renamed never to Nop, since this is a more precise name
I have also added a comment describing the purpose

**Pull request merged**